src/include/kdbchangetracking.h

```diff
@@ -0,0 +1,32 @@
+#ifndef ELEKTRA_KDBCHANGETRACKING_H
+#define ELEKTRA_KDBCHANGETRACKING_H
+
+#include <kdb.h>
+#include <kdbdiff.h>
+#include <kdbplugin.h>
+#include <kdbtypes.h>
+
+#ifdef __cplusplus
+namespace ckdb
+{
+extern "C" {
+#endif
+
+typedef struct _ChangeTrackingContext ChangeTrackingContext;
+
+const ChangeTrackingContext * elektraChangeTrackingGetContextFromKdb (KDB * kdb) ELEKTRA_WEAK;
+const ChangeTrackingContext * elektraChangeTrackingGetContextFromPlugin (Plugin * plugin) ELEKTRA_WEAK;
```

Comment:
As it turns out, we need to declare these two functions as `weak` on macOS, else the tests will not pass.